dist/betteR20-5etools.user.js

```diff
@@ -2786,7 +2788,31 @@ function baseTool () {
 						else return getAllJournalItems().filter(it => it);
 					}
 
+					// Allow for deleting tables as well
+					function getRollableTables() {
+						const tItems = [];
+						if ($cbTables.prop("checked")) {
+							// Get a tableobject from the d20 thing and loop through it
+							tableObject = d20.Campaign.rollabletables;
```

Comment:
Missing `const`

Reply:
Unsure why you're making this into a variable, anyway: use the full `d20.Campaign.rollabletables` everywhere instead. Don't rename things unless you have to--typing 10 extra characters is significantly easier than remembering function-specific aliases